Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Dockerfile for ACA-Py with all plugins
+ARG PYTHON_VERSION
+FROM python:${PYTHON_VERSION}-slim-bookworm
+
+# Redeclare build args for use in RUN commands
+ARG REPO_OWNER
+ARG ACA_PY_VERSION
+
+USER root
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    libsqlcipher-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install ACA-Py
+RUN pip install acapy-agent==${ACA_PY_VERSION}
+
+# Create acapy user
+RUN useradd -m -s /bin/bash acapy
+
+# Install plugins with --no-deps to avoid dependency conflicts
+RUN pip install --no-deps \
+    git+https://github.com/${REPO_OWNER}/acapy-plugins@${ACA_PY_VERSION}#subdirectory=basicmessage_storage \
```

Comment:
I think this works fine. I was trying to think if copying the source code and then installing would be better. I think I like doing it this way though. Much simpler.